src/main/java/com/mael/payroll/cards/TimeCard.java

```diff
@@ -0,0 +1,19 @@
+package com.mael.payroll.cards;
+
+public class TimeCard {
+    private int date;
+    private double hours;
+
+    public TimeCard(int date, double hours) {
```

Comment:
Yeah, that's a simplification done in the book just to have things running, later on they will re-visit this with a proper `Date()` object.